app/addons/components/components/stringeditmodal.js

```diff
@@ -39,6 +40,7 @@ export class StringEditModal extends React.Component {
     this.editor = ace.edit(dom_node);
     this.editor.setShowPrintMargin(false);
     this.editor.setOption('highlightActiveLine', true);
+    this.editor.setOption('wrap', !!this.props.wordWrapEnabled);
```

Comment:
```suggestion
    this.editor.setOption('wrap', this.props.wordWrapEnabled);
```

Reply:
This is on purpose to turn falsy/truthy values to `false` / `true`